Makefile

```diff
@@ -131,27 +111,32 @@ core-update-api: assemble-core-release
 
 .PHONY: assemble-ui-debug
 assemble-ui-debug:
-	$(call run-gradle-tasks,$(UI_MODULES),assembleDebug) \
-	&& ./gradlew libnavui-dropin:assembleDebug
+	$(call run-gradle-tasks,$(UI_MODULES),assembleDebug)
 
 .PHONY: assemble-ui-release
 assemble-ui-release:
-	$(call run-gradle-tasks,$(UI_MODULES),assembleRelease) \
-	&& ./gradlew libnavui-dropin:assembleRelease
+	$(call run-gradle-tasks,$(UI_MODULES),assembleRelease)
 
 .PHONY: ui-unit-tests
 ui-unit-tests:
-	$(call run-gradle-tasks,$(UI_MODULES),test) \
-	&& ./gradlew libnavui-dropin:test
+	$(call run-gradle-tasks,$(UI_MODULES),test)
 
 .PHONY: ui-unit-tests-jacoco
 ui-unit-tests-jacoco:
-	$(call run-gradle-tasks,$(UI_MODULES),jacocoTestReport) \
-	&& ./gradlew libnavui-dropin:jacocoTestReport
+	$(call run-gradle-tasks,$(UI_MODULES),jacocoTestReport)
+
+.PHONY: publish-local
+core-publish-local:
+	./gradlew publishToMavenLocal
+
+.PHONY: upload-to-sdk-registry
```

Comment:
Not sure I understand all changes correctly, but looks like previously this task didn't consider `libnavui-dropin` module, but now it does. Is it expected?

Reply:
`libnavui-dropin` does not apply the `publish.gradle` script so the publishing tasks are not available for that module.